cumulusci/cumulusci.yml

```diff
@@ -650,6 +656,7 @@ flows:
                     managed: True
 
     config_apextest:
+        group: Org Configuraiton 
```

Comment:
Let's call this group Post-Install Configuration, because that makes it clearer that it's meant to be run after the package is installed. (also note misspelled configuration)